.eslintrc.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
     '@babel/no-unused-expressions': 'off',
     'import/named': 'off',
     'import/no-default-export': ['error'],
+    'import/no-extraneous-dependencies': ['error', {packageDir: ['../', './']}],
```

Comment:
@BPScott @aaronccasanova what do you think about this?

Reply:
I think some cache is hiding errors for you

`yarn eslint ./polaris-react` in the root of the repo causes thouandss of errors for me. Sample:

```
/Users/ben/src/github.com/Shopify/polaris/polaris-react/tests/utilities/react-testing.tsx
0:1  error  The package.json file could not be found                                                                                  import/no-extraneous-dependencies
  1:1  error  'react' should be listed in the project's dependencies. Run 'npm i -S react' to add it                                    import/no-extraneous-dependencies
  2:1  error  '@shopify/react-testing' should be listed in the project's dependencies. Run 'npm i -S @shopify/react-testing' to add it  import/no-extraneous-dependencies
```